### ISSUE TITLE
Allow for unnamed requests and display the url as the name

### ISSUE THREAD

**Issue by Jebarson** [getinsomnia/insomnia#7970]
### Expected Behavior

The issue specified in https://github.com/Kong/insomnia/issues/1505 is still not available. I should be seeing an experience as what is mentioned in https://github.com/Kong/insomnia/pull/2230

### Actual Behavior

I am still seeing everything as "New Request"

### Reproduction Steps

Create a new request using the + symbol and dont name it

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

10.0.0

### What operating system are you using?

Windows

### Operating System Version

Win 11

### Installation method

download from portal

**Comment by subnetmarco** [comment]
Yes, I think this is a very good idea indeed. Let me see if it can be quickly implemented.
